common-rendering/src/components/keyEvents.tsx

```diff
@@ -0,0 +1,187 @@
+import { FC } from "react";
+import { textSans, headline } from "@guardian/src-foundations/typography";
+import { palette, remSpace } from "@guardian/src-foundations";
+import { SvgChevronUpSingle, SvgChevronDownSingle } from "@guardian/src-icons";
+import { Pillar, Theme } from "@guardian/types";
+import { from } from "@guardian/src-foundations/mq";
+
+// ----- Component ----- //
+type paletteId = 300 | 400;
+
+interface KeyEvent {
+	time: string;
+	text: string;
+	url: string;
+}
+
+interface KeyEventsProps {
+	keyEvents: KeyEvent[];
+	theme: Theme;
+}
+
+interface ListItemProps {
+	keyEvent: KeyEvent;
+	theme: Theme;
+}
+
+const getColor = (theme: Theme, paletteId: paletteId) => {
+	switch (theme) {
+		case Pillar.Sport:
+			return palette.sport[paletteId];
+		case Pillar.Culture:
+			return palette.culture[paletteId];
+		case Pillar.Lifestyle:
+			return palette.lifestyle[paletteId];
+		case Pillar.Opinion:
+			return palette.opinion[paletteId];
+		default:
+			return palette.news[paletteId];
+	}
+};
+
+const keyEventWrapperStyles: SerializedStyles = css`
+	width: 100%;
+
+	${from.desktop} {
+		width: 13.75rem;
+	}
+`;
+
+const detailsStyles: SerializedStyles = css`
+	&:not([open]) .is-on,
+	&[open] .is-off {
+		display: none;
+	}
+
+	summary::-webkit-details-marker {
+		display: none;
+	}
+`;
+
+const titleRowStyles: SerializedStyles = css`
+	position: relative;
+	margin: ${remSpace[1]} ${remSpace[1]} 0 ${remSpace[3]};
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+
+	&:focus {
+		outline: none;
+	}
+
+	path {
+		fill: ${palette.neutral[46]};
+	}
+
+	svg {
+		height: 2rem;
+	}
+
+	${from.phablet} {
+		margin: ${remSpace[1]} ${remSpace[4]} 0;
+	}
+
+	${from.desktop} {
+		display: none;
+	}
+`;
+
+const titleStyle: SerializedStyles = css`
+	${headline.xxsmall({ fontWeight: "bold", lineHeight: "tight" })};
+	color: ${palette.neutral[7]};
+`;
+
+const listStyles: SerializedStyles = css`
+	margin: ${remSpace[3]};
+
+	${from.phablet} {
+		margin: ${remSpace[3]} ${remSpace[5]} 0;
+	}
+
+	${from.desktop} {
+		margin: ${remSpace[1]} 0 0;
+	}
+
+	li::before {
+		content: "";
+		border-color: transparent ${palette.neutral[7]};
+		border-style: solid;
+		border-width: 0.4rem 0 0.4rem 0.5rem;
+		display: block;
+		height: 0;
+		width: 0;
+		top: 0;
+		position: absolute;
+	}
+`;
+
+const listItemStyles: SerializedStyles = css`
+	padding-bottom: ${remSpace[3]};
+	border-left: 1px solid ${palette.neutral[7]};
+	position: relative;
+
+	&:last-child {
+		border-left: none;
+	}
+`;
+
+const timeTextWrapperStyles: SerializedStyles = css`
+	margin-left: ${remSpace[4]};
+`;
+
+const textStyles = (theme: Theme): SerializedStyles => css`
+	${headline.xxxsmall({ fontWeight: "regular", lineHeight: "regular" })};
+	color: ${getColor(theme, 300)};
+	text-decoration: none;
+`;
+
+const timeStyles: SerializedStyles = css`
+	${textSans.xxsmall({ fontWeight: "bold", lineHeight: "tight" })};
+	color: ${palette.neutral[7]};
```

Comment:
There's `text.anchorSupporting` that might be better here?

https://theguardian.design/2a1e5182b/p/1377a6-tokens/t/85d3b0

Reply:
anchorSupporting doesn't seem to exist for text. The color I need is #121212 which exists in `text.primary`, `text. anchorSecondary`, `text.userInput`, `text.inputLabel` & `text.groupLabel`

@oliverlloyd Do you think any of these be used for the `time` of the key events?

Reply:
Oh no, the docs are maybe a bit off. Hopefully this will be resolved when everything moves to Storybook.

To be honest, I find it difficult to marry all the various use cases on a page with the list of token names available in Source. This was one of the reasons we created that `decidePalette` thing, it lets us use our own names!

I'm happy if you want to choose what you feel is best here. Be that one of the tokens or to use `neutral` directly. In general, I don't think we need to be dogmatic about using named tokens, it's just nice to think it through